src/components/views/BlackCardSelection.js

```diff
@@ -72,22 +114,41 @@ const BlackCardSelection = () => {
                 setCards(response.data)
             }
             catch (error) {
-                // todo if user already has selected a Card, server should probably throw an error
                 console.error("Details:", error);
                 alert("Invalid Input:\n " + handleError(error));
             }
         }
+        fetchBlackCard();
         fetchCards();
+
     }, []);
 
+    // placeholder in case of failure
+    let textContent;
+    let cardContent;
+    let now = new Date();
+    const diffTime = Math.abs(Date.parse(now) - Date.parse(startTime));
 
+    const diffTimeAsString = msToHM(diffTime);
 
-    // placeholder in case of failure
-    let content = <div>No Content Available</div>
+    if(blackCard!==null){
+        textContent =
+        <div className={"game description"}>
+            <h1>You have already chosen a black card</h1>
+            <h2>time played on this active black card: {diffTimeAsString}</h2>
```

Comment:
That looks kinda ugly. I would propose just writing smth like: You can choose a new black card in {23:43:22} (This would be nice if you could add a countdown (+ you have the perks of knowing how to do a countdown afterwards). I think the react tutorial has a countdown, so that would be nice.

Reply:
done